package.json

```diff
@@ -134,7 +134,7 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test --env=./custom-test-env.js",
-    "test:e2e": "start-server-and-test 'serve build -l 3000' http://localhost:3000 'cypress run'",
+    "test:e2e": "start-server-and-test 'serve build -l 3000' http://localhost:3000 'cypress run --record'",
```

Comment:
accident? you can add record argument to the ci job if you use this script locally a lot

Reply:
I must have accidentally removed the `--record` from ci in #3094. I'm fine removing it (as we don't pay for recording), but this is just reverting it to what it was.

Reply:
feel free to remove it, but should make the corresponding change to the ci (add `--record` [here](https://github.com/Uniswap/interface/blob/aa97ec01d311b8e2adb8dc736896f12ccf47b044/.github/workflows/integration-tests.yaml#L47))